lib/src/realm_class.dart

```diff
@@ -77,8 +70,22 @@ export "configuration.dart"
         RealmSchema,
         SchemaObject,
         ShouldCompactCallback,
+        SyncError,
+        SyncClientError,
+        SyncConnectionError,
+        SyncSessionError,
+        GeneralSyncError,
+        GeneralSyncErrorCode,
+        ClientResetError,
         SyncErrorHandler,
-        SyncClientResetErrorHandler;
+        ClientResetHandler,
+        ManualRecoveryHandler,
+        DiscardUnsyncedChangesHandler,
+        RecoverUnsyncedChangesHandler,
+        RecoverOrDiscardUnsyncedChangesHandler,
+        BeforeResetCallback,
+        AfterResetCallback,
+        ClientResetCallback;
```

Comment:
Should we perhaps just sort all of these alphabetically?

Reply:
I don't think they are grouped by types: erros, callbacks, handlers...

Reply:
if we decide to do that it will be easy since vs code has that command

Reply:
I like the classes for similar things to be side by side. Is it a common practice to be alphabetically?

Reply:
Done